source/whats-new.txt

```diff
@@ -41,6 +41,10 @@ What's New in 5.0
 
 New features of the 5.0 {+driver-short+} release include:
 
+- By default, the driver no longer checks types referenced in dot notation
+  unless the ``StrictFilter`` feature is explicitly
+  implemented. To learn more about this change, see the :ref:`Typescript fundamentals page <node-ts-type-safety>`.
+
 - Optional installation of ``@aws-sdk/credential-providers`` as a dependency.
 
   - The driver no longer includes AWS SDK modules by default. Use the
```

Comment:
Sorry this is unrelated to the work here, the aws sdk version below isn't correct, and some shells don't work well with `@` and caret symbols `^` (which we should also add to the version) so quoting it is a good idea for maximum compat:

```bash
npm install --save "@aws-sdk/credential-providers@^3.201.0"
```